pkg/cri/sbserver/container_create.go

```diff
@@ -1007,6 +1039,8 @@ func (c *criService) linuxContainerMounts(sandboxID string, config *runtime.Cont
 			HostPath:       sandboxDevShm,
 			Readonly:       false,
 			SelinuxRelabel: sandboxDevShm != devShm,
+			//UidMappings:    uidMappings,
```

Comment:
you can add TODO here to explain why we don't support devShm

Reply:
Yes, this is part of the commit "WIP". I'll probably just remove those lines, it was to debug the CI :)